packages/server/src/utils/youtube.ts

```diff
@@ -103,7 +98,7 @@ export async function uploadToYouTube(
           defaultAudioLanguage: 'en',
         },
         status: {
-          privacyStatus: event[0].unlisted ? 'unlisted' : 'public',
+          privacyStatus: session.published ? 'public' : 'unlisted',
```

Comment:
Why were we previously setting the status based on event and now based on session?